server/models/study.py

```diff
@@ -176,6 +186,54 @@ def getImages(self, study):
             'meta.studyId': study['_id']}).distinct('meta.imageId')
         return Image().find({'_id': {'$in': imageIds}})
 
+    def addWaitingUser(self, study, user):
+        """
+        Add user to list of users waiting to participate in the study.
+        """
+        waitingUsers = study['meta'].get('waitingUsers', [])
+        waitingUsers.append(user['_id'])
+
+        # "setMetadata" will always save
+        self.setMetadata(
+            folder=study,
+            metadata={
+                'waitingUsers': waitingUsers
+            }
+        )
+
+    def removeWaitingUser(self, study, user):
+        """
+        Remove user from list of users waiting to participate in the study.
+        """
+        try:
+            waitingUsers = study['meta'].get('waitingUsers', [])
+            waitingUsers.remove(user['_id'])
+
+            # "setMetadata" will always save
+            self.setMetadata(
+                folder=study,
+                metadata={
+                    'waitingUsers': waitingUsers
+                }
+            )
+        except ValueError:
```

Comment:
Can you scope the associated `try` block for this `ValueError` to a more specific statement?